rootless/specconv/specconv_linux.go

```diff
@@ -0,0 +1,38 @@
+package specconv
+
+import (
+	"io/ioutil"
+	"strconv"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+// ToRootless converts spec to be compatible with "rootless" runc.
+// * Remove cgroups (will be supported in separate PR when delegation permission is configured)
+// * Fix up OOMScoreAdj
+func ToRootless(spec *specs.Spec) error {
+	return toRootless(spec, getCurrentOOMScoreAdj())
+}
+
+func getCurrentOOMScoreAdj() int {
+	b, err := ioutil.ReadFile("/proc/self/oom_score_adj")
+	if err != nil {
```

Comment:
Are these worth logging as a warning, or no need bothering? (I'm ok either way)

Reply:
I think no need to do logging, because cgroups are ignored either way